pkgs/racket-doc/scribblings/reference/equality.scrbl

```diff
@@ -11,16 +11,46 @@ preferred for most use cases.
 
 @defproc[(equal? [v1 any/c] [v2 any/c]) boolean?]{
 
- Two values are @racket[equal?] if and only if they are @racket[eqv?],
- unless otherwise specified for a particular datatype.
-
- Datatypes with further specification of @racket[equal?] include
- strings, byte strings, pairs, mutable pairs, vectors, boxes, hash
- tables, and inspectable structures. In the last six cases, equality
- is recursively defined; if both @racket[v1] and @racket[v2] contain
- reference cycles, they are equal when the infinite unfoldings of the
- values would be equal. See also @racket[gen:equal+hash] and
- @racket[prop:impersonator-of].
+ Determines whether @racket[v1] is ``equal to'' @racket[v2], in a manner that
```

Comment:
I think we can improve upon the use of scare quotes. How about "Determines whether v1 and v2 are the same values, in a specific meaning of the word ``same'', captured by the following general properties" and then give the properties. Then go on to discuss how this is actually determined in a per-datatype way (and have a link to the relevant prop for new ones).